waspc/src/Wasp/Generator/ServerGenerator/AuthG.hs

```diff
@@ -127,5 +127,7 @@ depsRequiredByAuth spec = maybe [] (const authDeps) maybeAuth
       AS.Dependency.fromList
         [ ("lucia", "^3.0.1"),
           ("@lucia-auth/adapter-prisma", "^4.0.0"),
-          ("oslo", "^1.1.2")
+          ("oslo", "^1.1.2"),
+          -- Argon2 is used for hashing passwords.
+          ("@node-rs/argon2", "^1.8.3")
```

Comment:
You're importing the dependency into the SDK, but listing the dependency under the server's package.json.

Reply:
Great catch, let me update it 👍

Reply:
It's installed in both places because of this line: https://vscode.dev/github/wasp-lang/wasp/blob/miho-replace-oslo-password/waspc/src/Wasp/Generator/SdkGenerator.hs#L204

But, it's not imported in the `server` directly, so that's redundant. I'll move it to SDK.